src/event.js

```diff
@@ -286,7 +286,7 @@
       connect.getLog().trace("Publishing event: %s", eventName).sendInternalLogToServer();
     }
 
-    if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data.contactId && !(data instanceof connect.Contact)) {
+    if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data !== null && data.contactId && !(data instanceof connect.Contact)) {
```

Comment:
Any reason that this isn't just `if (eventName.startsWith(connect.ContactEvents.ACCEPTED) && data && data.contactId && !(data instanceof connect.Contact)) {`

Reply:
Nope! I tried to keep it consistent with existing code and there were several instances of using `!== null` like [this](https://github.com/amazon-connect/amazon-connect-streams/blob/master/src/aws-client.js#L5179) and [this](https://github.com/amazon-connect/amazon-connect-streams/blob/master/src/aws-client.js#L6796). I'm happy to change it though 😀
WDYT?

Reply:
A fair point, I think `&& data &&` is cleaner. But I appreciate that you are trying to keep the codebase consistent, and it doesn't do any harm, so I'm going to approve.